en-US/firefox/firefox-quantum.lang

```diff
@@ -0,0 +1,112 @@
+## NOTE: Demo page at https://www-demo3.allizom.org/firefox/firefox-quantum/
+## URL: https://www-demo3.allizom.org/%LOCALE%/firefox/firefox-quantum/
+
+
+# HTML page title
+;2x faster and 30% less memory
+2x faster and 30% less memory
+
+
+# HTML page decription
+;Firefox Browser
+Firefox Browser
+
+
+;Download now to start browsing the fastest Firefox ever.
+Download now to start browsing the fastest Firefox ever.
+
+
+;<strong>Firefox</strong> Quantum
+<strong>Firefox</strong> Quantum
+
+
+;New. Fast. For good.
+New. Fast. For good.
+
+
+;Download Now
+Download Now
+
+
+;2x faster
+2x faster
+
+
+;Powered by a new, cutting-edge engine, Firefox has <a href="%(url)s">doubled its speed</a> from last year. Because the Internet waits for no one.
+Powered by a new, cutting-edge engine, Firefox has <a href="%(url)s">doubled its speed</a> from last year. Because the Internet waits for no one.
+
+
+;Lean, mean speed machine
+Lean, mean speed machine
+
+
+;Firefox Quantum’s new engine uses <a href="%(url)s">30%% less memory</a> than Chrome, so other programs won’t slow down during browsing. Now that’s a win-win.
+Firefox Quantum’s new engine uses <a href="%(url)s">30%% less memory</a> than Chrome, so other programs won’t slow down during browsing. Now that’s a win-win.
+
+
+;Beautiful, intelligent design
+Beautiful, intelligent design
+
+
+;Bookmarks + Pocket
+Bookmarks + Pocket
+
+
+;New Tab
+New Tab
+
+
+;Hello, gorgeous! Firefox’s sleek, new look comes loaded with intuitive features like in-browser screenshots and more.
+Hello, gorgeous! Firefox’s sleek, new look comes loaded with intuitive features like in-browser screenshots and more.
+
+
+;Get the job done… faster
+Get the job done… faster
+
+
+;Search across multiple sites, view your top pages and discover new content.
+Search across multiple sites, view your top pages and discover new content.
+
+
+;Library
+Library
+
+
+;Enjoy everything you’ve saved to Firefox while browsing in one, easy place.
+Enjoy everything you’ve saved to Firefox while browsing in one, easy place.
+
+
+;Extensions
+Extensions
+
+
+;Personalize Firefox with your favorite extras that help you do you.
+Personalize Firefox with your favorite extras that help you do you.
```

Comment:
I wish I knew what he hack this means…

Reply:
LOL!  "help you do you" means help you make what you want to be or become.